back/spec/acceptance/users_spec.rb

```diff
@@ -390,6 +390,21 @@
           expect(json_response[:data].pluck(:id)).to match_array group_users.map(&:id)
         end
 
+        example 'Search for users in group' do
```

Comment:
Do we want to include this example in the API docs?

Reply:
I agree, it's not ideal. I'm going to push this out as it is to get it fixed and open a ticket to refactor this as suggested later - maybe a little concerned that we would be making the queries more inefficient.